src/order/succ_pred.lean

```diff
@@ -103,6 +103,9 @@ lemma succ_mono : monotone (succ : α → α) := λ a b, succ_le_succ
 lemma lt_succ_of_not_maximal {a b : α} (h : a < b) : a < succ a :=
 (le_succ a).lt_of_not_le (λ ha, maximal_of_succ_le ha h)
 
+lemma covers_succ_of_not_maximal {a b : α} (h : a < b) : a ⋖ succ a :=
+⟨lt_succ_of_not_maximal h, λ c hc, (succ_le_of_lt hc).not_lt⟩
```

Comment:
Should we have dot notation aliases? Like `has_lt.lt.lt_succ` and `has_lt.lt.covers_succ`.

Reply:
That sounds nice. But maybe I should state those lemmas as `∃ b, a < b`? Does it matter?